README.md

```diff
@@ -69,6 +69,10 @@ This setup is adapted for local scenario and works without external DNS provider
 
 Consult with [local playground](/docs/local.md) documentation to learn all the details of experimenting with local setup.
 
+## Upgrade
+
+See [upgrade notes](./docs/upgrade.md)
```

Comment:
No, I mean general upgrade notes for future. So this is efectively first entry for that.

Reply:
But we don't have it yet, so it would be good to at least create empty placeholder `./docs/upgrade.md` document, otherwise this leads to broken link user experience right on main project page.

Reply:
@somaritane right, I forgot to add upgrade.md, added